src/components/ModelTableList/StatusGroup.js

```diff
@@ -128,15 +132,8 @@ function generateModelTableDataByStatus(groupedModels, activeUser) {
           // We're not currently able to get the controller name from the API
           // so display the controller UUID instead.
           {
-            content: (
-              <a
-                href="#_"
-                className="p-link--soft"
-                title={getStatusValue(model.info, "controllerUuid")}
-              >
-                {getStatusValue(model.info, "controllerUuid").split("-")[0] +
-                  "..."}
-              </a>
+            content: generateControllerUUID(
+              getStatusValue(model.info, "controllerUuid")
```

Comment:
Yeah because of the differences in the table layouts this was the simplest way forward but we could definitely look at ways to simplify it further. That would be a follow-up as you mentioned.